source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -286,7 +293,18 @@ export abstract class CardElement extends CardObject {
             },
             this.separatorOrientation);
 
-        return renderedSeparator;
+            if (AdaptiveCard.alwaysBleedSeparators && renderedSeparator && this.separatorOrientation == Enums.Orientation.Horizontal) {
```

Comment:
Nitnit: quick comment here to explain what this is condition is implying?

Reply:
Added a comment.